### PR TITLE
meaningful changes

### PR DESCRIPTION
**Pull request by tdib**
This change has been on my mind for quite some time, and I think it's about time I pulled the trigger and did it. I've been really nervous about doing this so please don't shut me down here. I want you to truly consider the impact this could make on society, and why it could benefit you and your family. Skibidi toilet is something that has impacted and influenced not only me, but my family and friends, and the children of our generation. These children will grow into young adults having been influenced by their childhood. Just imagine if they see this while they are young, they have their whole future ahead of them!

In conclusion, I propose these changes for the betterment of society, and I kindly ask that you consider them.

If you'd like to discuss further, please reach out to me on telegram or whatsapp.

Thank you.

### CODE REVIEW

**Review by StefanPej — APPROVED** [review]
Oh
Oh
Oh oh oh
Yeah
You have so many relationships in this life
Only one or two will last
You go through all the pain and strife
Then you turn your back and they're gone so fast
Oh yeah
And they're gone so fast, yeah
Oh, so hold on the ones who really care
In the end they'll be the only ones there
And when you get old and start losing your hair
Can you tell me who will still care?
Can you tell me who will still care?
Oh care
Mmmbop, ba duba dop
Ba du bop, ba duba dop
Ba du bop, ba duba dop
Ba du, oh yeah
Mmmbop, ba duba dop
Ba du bop, ba du dop
Ba du bop, ba du dop
Ba du, yeah
Said oh yeah
In an mmmbop they're gone
Yeah yeah
Yeah yeah
Plant a seed, plant a flower, plant a rose
You can plant any one of those
Keep planting to find out which one grows
It's a secret no one knows
It's a secret no one knows
Oh, no one knows
Mmmbop, ba duba dop
Ba du bop, ba duba dop
Ba du bop, ba duba dop
Ba du, oh yeah
Mmmbop, ba duba dop
Ba du bop, ba du dop
Ba du bop, ba du dop
Can you tell me? Oh
No, you can't 'cause you don't know
Can you tell me?
You say you can but you don't know
Can you tell me? Oh (which flower's going to grow?)
No, you can't 'cause you don't know
Can you tell me? (If it's going to be a daisy or a rose?)
You say you can but you don't know
Can you tell me? Oh (which flower's going to grow?)
No, you can't 'cause you don't know
Can you tell me?
You say you can but you don't know
Oh oh
You say you can but you don't know
You don't know, you don't know, oh
Mmmbop, ba duba dop
Ba du bop, ba duba dop
Ba du bop, ba duba dop
Oh yeah
Mmmbop, ba duba dop
Ba du bop, ba du dop
Ba du bop, ba du dop
Yeah
